Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://gems.ruby-china.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.1'
+gem 'rails', '4.2.6'
```

Comment:
not only change the Gemfile, should apply the changes using `http://railsdiff.org/4.2.1/4.2.6`